build/push-aws.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Push cockroach binaries to AWS.
+# This is run by circle-ci after successful docker push.
+# Requisites:
+# - circleci must have AWS credentials configured
+# - AWS credentials must have S3 write permissions on the bucket
+# - the aws cli must be installed on the machine
+# - the region must be configured
+
+set -eux
+
+BUCKET_PATH="cockroachdb/bin"
+LATEST="LATEST"
+
+now=$(date +"%Y%m%d-%H%M%S")
+today=$(echo ${now} | cut -d '-' -f1)
+binary_name="cockroach.${now}.${CIRCLE_SHA1}"
+
+# Fetch name of most recent binary.
+# TODO(marc): some versions of `aws s3 cp` allow - for stdin/stdout, but not all.
+tmpfile=$(mktemp /tmp/cockroach-push.XXXXXX)
+# Don't exit on errors, clearing the latest file is an easy way to force multiple uploads.
+set +e
+time aws s3 cp s3://${BUCKET_PATH}/${LATEST} ${tmpfile}
```

Comment:
rather than set +e and then -e, you can `time aws s3 cp s3://${BUCKET_PATH}/${LATEST} ${tmpfile} || true`